pkg/backend/aio_test.go

```diff
@@ -153,9 +158,7 @@ func TestBackEnd_CreateAioVolume(t *testing.T) {
 }
 
 func TestBackEnd_UpdateAioVolume(t *testing.T) {
-	testAioVolumeWithName := server.ProtoClone(&testAioVolume)
-	testAioVolumeWithName.Name = testAioVolumeName
-	t.Cleanup(server.CheckTestProtoObjectsNotChanged(testAioVolumeWithName)(t, t.Name()))
+	t.Cleanup(server.CheckTestProtoObjectsNotChanged(&testAioVolumeWithName)(t, t.Name()))
```

Comment:
I think a better approach is to add `test*WithName` variables to `checkGlobalTestProtoObjectsNotChanged` lists located in `backend_test.go` and `middleend_test.go`. Thus, those global vars are checked in all tests (in `checkGlobalTestProtoObjectsNotChanged`)

Reply:
Good suggestion, adding